install/linux/usr/share/odemis/sim/sparc2-ek-sim.odm.yaml

```diff
@@ -258,7 +258,9 @@ SPARC2: {
        mag: 0.31, # ratio
        na: 0.2, # ratio, numerical aperture
        ri: 1.0, # ratio, refractive index
-       pole_pos: [104, 104], # (px, px), position of the pole (aka the hole in the mirror)
+       pole_pos: [122, 145],  # (px, px), position of the pole (aka the hole in the mirror)
+       mirror_pos_top: [111, -64086007],  # (px, px/m),position of the bottom on the mirror depending on the wavelength
```

Comment:
add a space between `(px, px/m),` and `position`

I'm not sure what the sentence means `position of the bottom on the mirror depending on the wavelength`, or should it be `position of the bottom of the mirror depending on the wavelength`